parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -814,11 +814,11 @@ configGPUDrivers() {
     if [ "$OS" = "$UBUNTU_OS_NAME" ]; then
         mkdir -p /opt/{actions,gpu}
         if [ "${CONTAINER_RUNTIME}" = "containerd" ]; then
-            # if target cloud is AzureUSGovernmentCloud, and NVIDIA_DRIVER_IMAGE contains GRID then use the specific 535 NVIDIA driver image
+            # if target cloud is AzureUSGovernmentCloud or AzureChinaCloud, and NVIDIA_DRIVER_IMAGE contains GRID then use the specific 535 NVIDIA driver image
             # this is being added because of the host driver in Fairfax is not compatible with the 550 guest drivers now
             # It can be removed and synced with 550/other clouds, once it is moved to a compatible host driver (check with the HPC team)
-            if [ "$TARGET_CLOUD" = "AzureUSGovernmentCloud" ] && \
-            [ "${NVIDIA_DRIVER_IMAGE#*GRID}" != "$NVIDIA_DRIVER_IMAGE" ]; then
+            if { [ "$TARGET_CLOUD" = "AzureUSGovernmentCloud" ] || [ "$TARGET_CLOUD" = "AzureChinaCloud" ]; } && \
```

Comment:
Have you tested the curly bracket ? Just time I see that.

Reply:
+1  usually shellcheck will ask you to change "$TARGET_CLOUD" to "${TARGET_CLOUD}"